db/migrate/20151011144512_create_emaillinks.rb

```diff
@@ -0,0 +1,18 @@
+class CreateEmaillinks < ActiveRecord::Migration
+  def change
+    create_table :emaillinks do |t|
+      t.datetime :accept_or_rejected_at
+      t.boolean :accepted
+      t.integer :cnt_title_words
+      t.string :email_subject
+      t.string :from_email
+      t.string :from_name
+      t.string :url
+      t.string :title
+      t.integer :label_id
+      t.integer :user_id
```

Comment:
The :title, :url, :from_name, :from_email and :email_subject columns should not be allowed to be null